.github/workflows/main.yml

```diff
@@ -18,13 +18,11 @@ jobs:
       - name: Setup ROS
         uses: ros-tooling/setup-ros@master
         with:
-          required-ros-distributions: foxy
+          required-ros-distributions: galactic
       - name: Install Iceoryx Dependencies
         run: sudo apt-get update && sudo apt-get install -y cmake libacl1-dev libncurses5-dev pkg-config
       - name: Build & Test
         uses: ros-tooling/action-ros-ci@master
         with:
           package-name: rmw_iceoryx_cpp iceoryx_ros2_bridge
-          target-ros2-distro: foxy
-          vcs-repo-file-url: |
-            ./.github/workflows/additional_repos.repos
```

Comment:
Are you sure that we do not need the additional repos?

Reply:
Yes, we don't need it anymore. Since galactic iceoryx is available in [ros2.repos](https://raw.githubusercontent.com/ros2/ros2/master/ros2.repos).